.stylelintrc

```diff
@@ -1,43 +1,44 @@
 {
+    "extends": "stylelint-config-standard-scss",
     "rules": {
-        "color-no-invalid-hex": true,
-        "font-family-no-duplicate-names": true,
-        "font-family-name-quotes": "always-where-recommended",
-        "function-name-case": "lower",
-        "function-url-no-scheme-relative": true,
-        "function-url-quotes": "always",
-        "number-no-trailing-zeros": true,
-        "length-zero-no-unit": true,
-        "unit-case": "lower",
-        "unit-no-unknown": true,
-        "property-case": "lower",
-        "property-no-unknown": true,
-        "keyframe-declaration-no-important": true,
-        "declaration-no-important": true,
-        "declaration-block-no-shorthand-property-overrides": true,
-        "declaration-block-single-line-max-declarations": 1,
-        "declaration-block-trailing-semicolon": "always",
-        "declaration-block-semicolon-newline-after": "always-multi-line",
-        "block-no-empty": true,
-        "selector-pseudo-class-no-unknown": true,
-        "selector-pseudo-element-no-unknown": true,
-        "selector-pseudo-element-case": "lower",
-        "selector-type-case": "lower",
-        "selector-type-no-unknown": true,
-        "selector-max-empty-lines": 0,
-        "media-feature-name-case": "lower",
-        "media-feature-name-no-unknown": [
+        "indentation": 4,
```

Comment:
Some notes of config changes for review / discussion:

- [string-quotes](https://stylelint.io/user-guide/rules/list/string-quotes): we were a bit all over the place here. I picked `"single"` as that's what we also use for JS?
- [declaration-empty-line-before](https://stylelint.io/user-guide/rules/list/declaration-empty-line-before). I set this to `"never"`, as this fits with our conventions.
- [color-function-notation](https://stylelint.io/user-guide/rules/list/color-function-notation). I set this to `"legacy"` for IE support.
- [alpha-value-notation](https://stylelint.io/user-guide/rules/list/alpha-value-notation). I set this to `"number"` as this seems to be what we most commonly use.
- [value-no-vendor-prefix](https://stylelint.io/user-guide/rules/list/value-no-vendor-prefix). I decided to allow vendor prefixes, as occasionally we may want to use them?
- [at-rule-empty-line-before](https://stylelint.io/user-guide/rules/list/at-rule-empty-line-before). I disabled this as we often use `@include` and `@import` nested closely together.
- [no-descending-specificity](https://stylelint.io/user-guide/rules/list/no-descending-specificity). This one is more for readability, but I disabled it as it's by far the rule we break most often. I'd be interested in thoughts from others here if it's something we should want to adopt (it could be a warning for now?).
- [value-keyword-case](https://stylelint.io/user-guide/rules/list/value-keyword-case). I added an ignore rule here for local font names.
- [no-duplicate-selectors](https://stylelint.io/user-guide/rules/list/no-duplicate-selectors). I added this as a warning for now, but it's something that I think is fixable in this PR if we wanted to adopt it?
- [max-line-length](https://stylelint.io/user-guide/rules/list/no-duplicate-selectors). I disabled this as we sometimes declare SVG icons as Sass variables?
- [font-family-no-missing-generic-family-keyword](https://stylelint.io/user-guide/rules/list/font-family-no-missing-generic-family-keyword). I added an ignore here for the font-awesome icon font.
- [declaration-block-no-redundant-longhand-properties](https://stylelint.io/user-guide/rules/list/declaration-block-no-redundant-longhand-properties). I added an ignore here for CSS grid properties, as personally I find the lonhand properties often more readable?